### DISCUSSION TITLE
WPF in Win10,The scrollbar  not work

### DISCUSSION THREAD

**Discussion by sonce** [cefsharp/CefSharp#4679] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

120.1.110.0

### Operating System

Windows 10

### Architecture

x64

### .Net Version

4.8

### Implementation

WPF

### Reproduction Steps

```xml
<cef:ChromiumWebBrowser Address="http://kindeditor.net/demo.php" />
```
the scroll shown,but click the scrollbar,or mouse wheel,it doesn't work.
click <a> link ,open new window,the new page's scroll work fine.

### Expected behavior

scrollbar should work

### Actual behavior

click the scrollbar,or mouse wheel,it doesn't work.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
Did you check the log file for errors? Or test with the [MinimalExample](https://github.com/cefsharp/CefSharp.MinimalExample)?

Quick test with the `MinimalExample` and scrolling is working as expected. I've not hand any other reports of issues.

**Comment by KYRIENIU1** [comment]
I also had this problem, and only after I resized the external window of Cefsahrp would it have the effect after I scrolled it
